apps/projects/app/components/Content/Issues.js

```diff
@@ -449,24 +459,32 @@ class Issues extends React.PureComponent {
     const downloadedRepos = {}
     let totalCount = 0
 
-    Object.keys(data).forEach(nodeName => {
-      const repo = data[nodeName]
+    if (data && data.node0) {
```

Comment:
It's inelegant in the sense that `flattenIssues` should only care about flattening issues, and not whether the issues are already loaded or not. The more appropriate place for this check is before calling `flattenIssues`, but the code looked like such a mess, that I decided to put it here.

What broke without this? For some reason, instead of adding new issues, the same issues were repeated over and over. Super weird. But in principle, you have to wait until data.node0 exists cause otherwise the API hasn't provided new data.